### ISSUE TITLE
Datepicker: selected date missing focus styling

### ISSUE THREAD

**Issue by sarahmonster** [WordPress/gutenberg#15929]
When tabbing through the datepicker, non-existent focus styling on the selected date makes it look as though focus has been lost:

![2019-05-30 23 58 39](https://user-images.githubusercontent.com/376315/58670480-2513b800-8337-11e9-8f35-969f6e823c1b.gif)

To reproduce:

1. Write a new post.
2. Open the sidebar and select the date to open calendar.
3. Press "Tab" 7 times to select the currently-selected date.
4. No focus indicator is shown around the current selection.

I'd recommend using a much louder focus style than the existing browser-default outline—probably matching the focus ring style used on other components.

**Comment by afercia** [comment]
Thank you for opening a specific issue 🙂 I think this is already covered in https://github.com/WordPress/gutenberg/issues/15278 though?

**Comment by sarahmonster** [comment]
Looks like that issue is referring to the colour contrast of the white text against the blue background for the selected date.

The issue here is that the selected date doesn't have focus styling (or any change to the styling to indicate focus) when it acquires keyboard focus. As a result, a user tabbing through can easily lose their place. 

#15962 should resolve the problem once merged.
